torch/_dynamo/decorators.py

```diff
@@ -132,6 +133,26 @@ def fn(a):
     return fn
 
 
+def allow_in_graph(fn):
+    """This function is deprecated in favor of :func:`unsafe_allow_in_graph()`
+
+    If you are trying to black-box a Python function for use with torch.compile
+    (aka something similar to torch.fx.wrap),
+    do not use this API. Instead, please create a custom operator:
+    https://docs.google.com/document/d/1_W62p8WJOQQUzPsJYa7s701JXt0qf2OfLub2sbkHOaU/edit
```

Comment:
See comments in other PR.  allow_in_graph == wrap, custom ops are a different thing.  Let's make the gdoc a webpage.